src/ImageSharp.Interactive/README.md

```diff
@@ -5,16 +5,16 @@
 Load the nuget pacakge and namespaces
 ```csharp --project
 using SixLabors.ImageSharp;
-using SixLabors.ImageSharp.Interactive;
+using SixLabors.ImageSharp.Processing;
 using SixLabors.ImageSharp.PixelFormats;
 ```
 
 Create and diplay an image
 
 ```csharp --project
-var image = new Image<Rgb24>(100, 100);
-image.Mutate(c => c.BackgroundColor(Color.AliceBlue));
-image.Display(); 
+var image = new Image<Rgba32>(100, 100);
```

Comment:
We should use the `using` keyword for all these examples since `Image` implements `IDIsposable`.

Reply:
This readme will be transformed to notebook on the fly, each code block will become a cell. if we do var using the image variable will be killed at each cell execution. plus I don't think that `var using` is supported in .NET Interactive (csharp scripting)

Reply:
Doesn't `Display()` simply Base64 encode the output though? 

I'll need a thorough explanation of how this works because I am incredibly uncomfortable and cannot possibly endorse a system that doesn't promote best practice.

Reply:
Display produces display events for .NET Interactive according to the formatting code registered. The formatter here creates an img tag and encodes the image.

This nuget package is intended to provide functionalities to users consuming ImageSharp in notebooks, not in applications. 

The read.me file on the root of the repo can be modified to make it vene clear, its intent is to provide a get started for notebook users, not to avoid promoting best practices. 

Using vat syntax is not supported at the moment in csharp scripting, using it could be very strange as variables do not go out of scope when are at top level in a cell.

Reply:
Then I'm obviously missing something that you need to explain.

There are multiple image instances created in this readme. None of which are disposed of to reclaim pooled memory. 

How is memory managed if not via `IDisposable`?

Reply:
As the notebook file is closed the environment is released and all memory is free at that point. The user has also the ability to use the disposable pattern or attach disposable objects to the kernel life span. Csharp scripting is a csharp dialect. In a notebook declaring a top level variable gives it the scope of the entire notebook life span

Reply:
@JimBobSquarePants what do you think? if possible I'd like to merge and publish the nuget package and use it instead of the one I have created.

Reply:
I think we should comment somewhere in the readme (_I'm assuming comments will be presented to the user_) that the lack of using statement represents non-standard usage of the library and that are due to environmental constraints.

Reply:
The readme states this is for use in notebooks. There is other documentation about the library and other pattern usage.
I am not sure the value I would add by telling the notebook users:
"do not dispose or use the `using var` pattern in notebooks but use it in classic applications".

If you still feel nervous about this pr and package let me know so, notebook users needs image sharp and I would like to have it available with proper support.

Reply:
@JimBobSquarePants should I close this PR?

Reply:
You’re missing the point. The whole purpose of notebooks is to allow an interactive demonstration of the API. They’re a learning resource. 

Therefore we need to ensure that the consumer of that learning resource understands that in the real world they should manage the lifecycle of the images. 

Simply saying (I’m paraphrasing here) “This is for notebooks” isn’t sufficient. It’s important to explain real world requirements. 

Remember consumers don’t have the underlying knowledge of the technology you have.

Reply:
I thought there was already plenty of documentation about ImageSharp usage and design. Notebooks can be a learning experience but this read me is about using your library in notebooks, not how to use it in application / service development in general. I am not getting the point of flooding a user with details that are not in the scope of this extension package and notebook scenarios.